### ISSUE TITLE
Tests fail: reinstallation of module fails due to the usearch8-dummy installed

### ISSUE THREAD

**Issue by greatfireball** [greatfireball/Run-Utax#7]
``` bash
localmachine:~/Run-Utax #cpanm Run-Utax-0.1.tar.gz 
--> Working on Run-Utax-0.1.tar.gz
Fetching file:///root/Run-Utax/Run-Utax-0.1.tar.gz ... OK
Configuring Run-Utax-0.1 ... OK
Building and testing Run-Utax-0.1 ... FAIL
! Installing Run-Utax-0.1.tar.gz failed. See /root/.cpanm/work/1439319425.26357/build.log for details. Retry with --force to force install it.
localmachine:~/Run-Utax #cat /root/.cpanm/work/1439319425.26357/build.log
[...]
t/01_script_compiles.......................ok
t/02_script_runs_without_parameters........ok
t/03_script_runs_with_help_requested.......ok
t/04_script_runs_with_man_requested........ok
t/05_script_runs_with_correct_arguments....ok
t/30_Run-Utax_loading......................ok
t/31_Run-Utax_run..........................ok
t/32_Run-Utax-check-parameters.............
#   Failed test 'Program dies, if no usearch is available'
#   at t/32_Run-Utax-check-parameters.t line 21.

#   Failed test 'usearch8 was found using the PATH evironmental variable'
#   at t/32_Run-Utax-check-parameters.t line 30.
#          got: '/usr/bin/usearch8'
#     expected: 'blib/script/usearch8'
# Looks like you failed 2 tests of 49.
dubious
    Test returned status 2 (wstat 512, 0x200)
DIED. FAILED tests 4, 6
    Failed 2/49 tests, 95.92% okay
t/33_Run-Utax-check-setter_getter..........ok
Failed Test                      Stat Wstat Total Fail  List of Failed
-------------------------------------------------------------------------------
t/32_Run-Utax-check-parameters.t    2   512    49    2  4 6
Failed 1/9 test scripts. 2/103 subtests failed.
Files=9, Tests=103,  2 wallclock secs ( 1.00 cusr +  0.13 csys =  1.13 CPU)
Failed 1/9 test programs. 2/103 subtests failed.
make: *** [test_dynamic] Fehler 255
```

**Comment by greatfireball** [comment]
Can be fixed using a different PATH for the tests

**Comment by greatfireball** [comment]
Reason for the behavior is addressed by issue #6 

Nevertheless, this issue should be fixed to avoid problems with real installed usearch8 programs
